pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -40,7 +43,25 @@
   - heroes_arranged.each_with_index do |entry, index|
     -# note: this gsheet display parameter unused due to IE8 fix.
     - display = entry[:active_s] == "hide" ? "none" : "block"
-    .widehero.lazyload{id: "fullwidth#{index}", "data-bg"=> "#{entry[:image_t]}", style: "background-position: #{entry[:param1_s]}"}
+    - image_2x = entry[:image_t].gsub('.jpg', '_2x.jpg')
+    - if index == 0
+      -# Use CSS media query to select hero image to preload/render first.
+      %link{rel: 'preload', href: image_2x, as: 'image', media:'(min-width: 1440px)'}
+      %link{rel: 'preload', href: entry[:image_t], as: 'image', media:'not all and (min-width: 1440px)'}
+      :css
+        @media (min-width: 1440px) {
+          #fullwidth#{index} {
+            background-image: url(#{image_2x});
+          }
+        }
+        @media not all and (min-width: 1440px) {
+          #fullwidth#{index} {
+            background-image: url(#{entry[:image_t]});
+          }
+        }
```

Comment:
I realize that you have it inlined here to generate the correct URLs for each image.  I'd really prefer we use the `srcset` attribute instead: https://stackoverflow.com/a/28731347.